includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -630,6 +630,39 @@ public function scripts() {
 		}
 	}
 
+	/**
+	 * Returns true if given product needs shipping. For bundled products it will return true if bundle is not virtual (assembled)
+	 * otherwise if it's virtual bundle it will check if at least one bundled item needs shipping in order to return true.
+	 *
+	 * @since   x.x.x
+	 * @version x.x.x
+	 * @return  boolean True if product needs shipping.
+	 */
```

Comment:
If we end up merging this, we should document in the comment section that this is a stop gap because `$product->needs_shipping()` doesn't work with the bundles extension.

And add a TODO on what needs to be done as cleanup if the extension fixes it.